make-rules/ips.mk

```diff
@@ -247,7 +247,7 @@ publish:		pre-publish $(PUBLISH_STAMP)
 
 sample-manifest:	$(GENERATED).p5m
 
-$(GENERATED).p5m:	install
+$(GENERATED).p5m:	build install
```

Comment:
I'm not sure that it's a good idea. Install should depend on build, that's all.

Reply:
Nor am I... I dropped the commit and will just go with the comment in the Makefile about the situation.